link/src/test/kotlin/com/meshconnect/link/utils/OnPageFinishedScriptBuilderTest.kt

```diff
@@ -0,0 +1,36 @@
+package com.meshconnect.link.utils
+
+import com.meshconnect.link.randomString
+import org.amshove.kluent.internal.assertEquals
+import org.junit.Test
+
+@Suppress("MaxLineLength")
+class OnPageFinishedScriptBuilderTest {
+    @Test
+    fun `test default constructor`() {
+        val version = randomString
+        val script = OnPageFinishedScriptBuilder(version).build()
+        assertEquals(
+            "window.meshSdkPlatform='android';window.meshSdkVersion='$version';if(window.parent) {window.parent.meshSdkPlatform='android';window.parent.meshSdkVersion='$version';}",
```

Comment:
mobile SDK's webview should not be in an iFrame, why do we need
`if(window.parent) {window.parent.meshSdkPlatform=...` ?

Reply:
Good point